vscode-dotnet-runtime-library/src/Acquisition/IDotnetCoreAcquisitionWorker.ts

```diff
@@ -14,9 +14,9 @@ export interface IDotnetCoreAcquisitionWorker
 {
     uninstallAll(eventStream : IEventStream, storagePath : string, extensionState : IExtensionState): void;
 
-    acquireRuntime(context: IAcquisitionWorkerContext, invoker : IAcquisitionInvoker): Promise<IDotnetAcquireResult>;
+    acquireLocalRuntime(context: IAcquisitionWorkerContext, invoker : IAcquisitionInvoker): Promise<IDotnetAcquireResult>;
```

Comment:
Should this have an `acquireLocalASPNET` method as well?

Reply:
I wrote the same comment on my review but discarded it... idk why, good point so Ill add it!